### DISCUSSION TITLE
Need a spped meter application for windows 11

### DISCUSSION THREAD

**Discussion by SjxSubham** [community/community#139133] [Programming Help]
### Body

I need a speed meter application which will show the downloading and uploading speed in the taskbar.. suggest+provide me some with software download links....

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by matheuslpiresm** [comment]
I've already used NetSpeedMonitor to do this

**Reply by SjxSubham** [reply]
but NetSpeedMonitor is only for windows 7/Xp/vista 
I want for windows 10/11

**Comment by matheuslpiresm** [comment]
Oh yeah, I found this one while doing my research, I hope it helps you BitMeter II

Link: https://codebox.net/pages/bitmeter2

**Comment by Xelliese** [comment]
Another good one is DU Meter, but it's paid.
